src/mmw/js/src/app.js

```diff
@@ -50,13 +51,17 @@ var App = new Marionette.Application({
     },
 
     showLoginModal: function() {
-        $('#login').find('#id_username').val("");
-        $('#login').find('#id_password').val("");
-        $('#login').modal('show');
+        var userView = new userViews.LoginModalView({
+            model: new userModels.LoginFormModel({}),
+            app: this
+        }).render();
+        userView.$el.find('#id_username').val('');
```

Comment:
Consider creating some helper methods so we don't have to manipulate `$el` outside of its view.